snuba/web/rpc/v1/resolvers/R_eap_spans/resolver_get_trace.py

```diff
@@ -86,6 +87,19 @@ def _build_query(request: GetTraceRequest) -> Query:
                 ),
             ),
         ]
+        columns_to_exclude = ["retention_days", "sign", "attr_str", "attr_num"]
```

Comment:
You want to add `span_id` and `timestamp` in there as well as we're already fetching them.